port_ocean/clients/port/client.py

```diff
@@ -75,3 +76,13 @@ async def get_org_id(self) -> str:
         handle_status_code(response)
 
         return response.json()["organization"]["id"]
+
+    async def update_integration_state(self, state: dict[str, Any]) -> dict[str, Any]:
+        logger.debug(f"Updating integration state with: {state}")
+        response = await self.client.patch(
+            f"{self.api_url}/integration/{self.integration_identifier}",
+            headers=await self.auth.headers(),
+            json={"state": state},
+        )
+        handle_status_code(response)
+        return response.json().get("integration", {})
```

Comment:
is this a blocker?
is it mandatory for this PR, because if we do have time to invest in this infra then I'd rather use a whole other infra set to save this data in